### ISSUE TITLE
[blocker] 'standard-minifiers' in api.use

### ISSUE THREAD

**Issue by juliancwirko** [orionjs/orion#348]
Hi, I wonder why there is a `standard-minifiers` dependency in Orion. This is a real blocker for users who want to use similar packages but not `standard-minifiers`. I mean for example [juliancwirko:postcss](https://github.com/juliancwirko/meteor-postcss) (autoprefixer usage and other postcss plugins). 

`standar-minifiers` is default, but it could be replaced by other packages and there should be only one added at the same time (only one js and css minifier registered). So in my opinion there shouldn't be such dependency in community based packages because it will block Meteor app at startup. There will be no possibility to remove `standard-minifiers`.

Related discussion: https://forums.meteor.com/t/postcss-package-and-meteor-build-plugin-questions/12454/13

**Comment by JohnnyRuvac** [comment]
+ 1

**Comment by macrozone** [comment]
Can we just remove this dependency in the package.js? If yes, can you start a PR for this?

**Comment by juliancwirko** [comment]
Yes, I think that there shouldn't be a dependency because every app will have standard-minifiers and if someone will want to replace it in the app it will be able to do it. Is there a need for PR? This is just one line deletion.

**Comment by macrozone** [comment]
Any progress on that? That is really a big problem as you cannot use e.g. `seba:minifiers-autoprefixer`  :-(
